aries_vcx/src/protocols/proof_presentation/verifier/verification_status.rs

```diff
@@ -0,0 +1,64 @@
+use serde::{Deserialize, Deserializer};
+
+#[derive(Serialize, Debug, Clone, PartialEq, Eq)]
+pub enum PresentationVerificationStatus {
+    Valid,
+    Invalid,
+    Unavailable(),
```

Comment:
I know this is removed, but I wanted to look it up. I didn't even know this is possible.
It's actually very interesting that you can declare variants like that, getting a constructor closure for free.

Nevertheless, in this case it is redundant. And yeah, the serialization -> deserialization roundtrip fails because the output changes. Apparently `serde` serializes this to `"Unavailable": []`, probably because in proc_macros tuple variants get treated as arrays of fields.

So apart from single field tuples, which are handled transparently, you end up with an array. Interestingly, I would expect the empty tuple variant to be set to `null` instead.